src/components/PageTitleCards.tsx

```diff
@@ -0,0 +1,20 @@
+interface Props {
+  mainText: string;
+  subText?: string;
+  primaryColor: string;
+}
+
+const PageTitleCards = ({ mainText, subText, primaryColor }: Props) => {
```

Comment:
can we rename this to "PageTitleCard"?

Reply:
"PageTittleTag" is okay? Following the rules of "Card = Tag"?

Reply:
Change made